playgrounds/next/src/app/test/media/page.tsx

```diff
@@ -0,0 +1,158 @@
+'use client';
+
+import { Media } from '@0xsequence/marketplace-sdk/react';
+import { useState } from 'react';
+
+// Mock assets for testing - using stable URLs that won't change
+const TEST_ASSETS = {
+	// Using a data URL for truly stable testing
+	image:
+		'data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAwIiBoZWlnaHQ9IjQwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8cmVjdCB3aWR0aD0iNDAwIiBoZWlnaHQ9IjQwMCIgZmlsbD0iI0ZGMDAwMCIvPgogIDx0ZXh0IHg9IjUwJSIgeT0iNTAlIiBmb250LWZhbWlseT0iQXJpYWwiIGZvbnQtc2l6ZT0iMjQiIGZpbGw9IndoaXRlIiB0ZXh0LWFuY2hvcj0ibWlkZGxlIiBkb21pbmFudC1iYXNlbGluZT0ibWlkZGxlIj5UZXN0IEltYWdlPC90ZXh0Pgo8L3N2Zz4=',
+	video:
+		'https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_1MB.mp4',
+	brokenImage: 'https://broken-url-that-will-fail.com/image.jpg',
+	iframe: 'https://www.youtube.com/embed/dQw4w9WgXcQ',
+	model3d: 'https://modelviewer.dev/shared-assets/models/Astronaut.gltf',
```

Comment:
This page gives 404, can you change it as following?


```suggestion
	model3d: 'https://modelviewer.dev/shared-assets/models/NeilArmstrong.glb',
```